### ISSUE TITLE
Shift Plugins

### ISSUE THREAD

**Issue by juliavisserinbibo** [Inbibo/shift_docs#17]
This issue addresses the content of the Shift Plugins page. This page should explain what are Shift Plugins, how to create them, and add them to Shift. This should include:

- Description of a Shift Plugin
- Where to find Plugins in the Shift UI and place them in your workspace
- Shift Native Plugins
   - Operators List
   - Variables Browser
- Walkthrough of the structure and requirements for creating a Shift Plugin
   - This should include a link to Shift’s API for plugins.
- How to add a Plugin to Shift through environment variables

Screenshots and new section pages should be added where necessary.

**Comment by albertochimentiinbibo** [comment]
PR opened

**Comment by albertochimentiinbibo** [comment]
Currently on Hold waiting for:

- https://github.com/Inbibo/shift/issues/1103#issue-2483230759 needed to re record the gif for the "Nodes List Plugin"
- https://github.com/Inbibo/shift/issues/1119#issue-2491578486 needed for the new getter method and changes to the pluginTemplate file (this one is also waiting on the previous one)
